src/containers/gui.jsx

```diff
@@ -43,6 +44,7 @@ class GUI extends React.Component {
                 // error page gets rendered if project failed to load
                 this.setState({loadingError: true, errorMessage: e});
             });
+        this.props.vm.initialized = true;
```

Comment:
This is a hack... we should figure out what the best way to ask the vm if it's already been started.  What we really want to know is if there's a project loaded / if the vm is running.  It looks like the most accurate would be `if (typeof this.props.vm.runtime._steppingInterval !== 'undefined')`... but I think we'd want to just make an API on `scratch-vm` for this purpose.